packages/block-library/src/navigation/index.php

```diff
@@ -133,8 +133,23 @@ function render_block_navigation( $content, $block ) {
 		return $content;
 	}
 
-	$attributes           = $block['attrs'];
 	$block['innerBlocks'] = core_block_navigation_empty_navigation_links_recursive( $block['innerBlocks'] );
+	$attributes           = $block['attrs'];
+
+	// Deprecated:
+	// The rgbTextColor and rgbBackgroundColor attributes
+	// have been deprecated in favor of
+	// customTextColor and customBackgroundColor ones.
+	// Move the values from old attrs to the new ones.
```

Comment:
Could we change that to using the `/*` (single asterisk) notation, as described in  https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#5-2-multi-line-comments

Reply:
of course, super glad to do it but is it possible in another PR? I don't want to lose the green flag of tests.

Reply:
Let's do it now. I'm happy to click the re-run button for you to get them green again :)